Tools/CISupport/ews-build/steps.py

```diff
@@ -1691,6 +1691,9 @@ def send_email_for_github_failure(self):
 class ValidateCommitterAndReviewer(buildstep.BuildStep, GitHubMixin, AddToLogMixin):
     name = 'validate-commiter-and-reviewer'
     descriptionDone = ['Validated commiter and reviewer']
+    VALIDATORS_FOR = {
+        'apple': ['geoffreygaren', 'markcgee'],  # FIXME: This should be a bot, for now validation is manual
```

Comment:
Do we want to hard-code the list here, or in `emails.json`?
Either way is fine with me, just wanted to point out an alternative.

Also, if this is going to be permanent, we can consider adding this info to `contributors.json` so that we don't have to maintain such lists at multiple places.

Reply:
Discussed this offline with @aj062.

We _do_ want this hard-coded, it's going to be a bot in the near future, which will make this hard-coding more natural than it is now.